src/tracking.c

```diff
@@ -394,10 +404,31 @@ void trackingInvalidateKeyRaw(client *c, char *key, size_t keylen, int bcast) {
     raxRemove(TrackingTable,(unsigned char*)key,keylen,NULL);
 }
 
-/* Wrapper (the one actually called across the core) to pass the key
- * as object. */
-void trackingInvalidateKey(client *c, robj *keyobj) {
-    trackingInvalidateKeyRaw(c,keyobj->ptr,sdslen(keyobj->ptr),1);
+void trackingScheduleKey(uint64_t client_id, robj *keyobj) {
+    invalidateKey *ik = zmalloc(sizeof(invalidateKey));
```

Comment:
A minor comment to make the intention of the function clearer.

```suggestion
void trackingScheduleKeyInvalidation(uint64_t client_id, robj *keyobj) {
    invalidatedKey *ik = zmalloc(sizeof(invalidatedKey));
```